vms/vms/urls.py

```diff
@@ -16,4 +21,13 @@
     url(r'^registration/', include('registration.urls', namespace='registration')),
     url(r'^shift/', include('shift.urls', namespace='shift')),
     url(r'^volunteer/', include('volunteer.urls', namespace="volunteer")),
+    url(r'^portal', TemplateView.as_view(template_name='home/home.html'),name='home'),
+    url(r'^login/$',
+         anonymous_required(auth_views.login),
+        {'template_name': 'authentication/login.html'},
```

Comment:
I see that the indentation is given as both single line and multi line. When it comes to password reset and email confirmation we can follow the same indentation.